tests/scope/good/191_fix_record_name_confusion.catala_en

```diff
@@ -25,6 +25,7 @@ $ catala Typecheck --check-invariants
 
 ```catala-test-inline
 $ catala OCaml -O
+Generating entry points for scopes: ScopeA ScopeB
```

Comment:
What is this line? A comment in the resulting `OCaml` file? A debug print by the compiler on `stdout`?

Reply:
Yes it's just a message... it was already there but got moved around :shrug:

Reply:
It was a rogue `eprintf`, I'll turn that into a debug.